php/ubiquity/README.md

```diff
@@ -0,0 +1,43 @@
+# Ubiquity Benchmarking Test
```

Comment:
could we remove this file ?
Using a `README` sounds like a `best-practice` but in this case it **NOT** relevant (all implementations **SHOULD** do the same thing)